src/js/pagination/specs/paging-collection-spec.js

```diff
@@ -89,6 +96,28 @@ define(['jquery',
                 }
             ));
 
+            it('doesn\'t send unnecessary parameters in request (total pages, total records)', AjaxHelpers.requests(
```

Comment:
because I have tried to use the english terms not the actual things what we have in code. And because `total_entries` could be other string. e.g. we could have `all_records` instead of `total_entries`.